index.html

```diff
@@ -58,6 +59,7 @@ <h2><i class="fas fa-calendar-alt"></i> Meeting Info</h2>
                     <p>As of November 2023, meetings are held on the 4th Tuesday evening of every month at 6 PM at Inland Packaging (north visitor entrance @ 3209 Airport Rd, La Crosse, WI 54603).</p>
                     <p>If you have trouble getting into the building or need directions, please give a shout on the Discord. Meeting cancellations, reschedulings, etc. will also be posted there.</p>
                     <p>We also post meeting info on the calendar at <a href="https://that.us/communities/la-crosse-area-software-developers/">https://that.us/communities/la-crosse-area-software-developers/</a></p>
+                    <p>To add the La Crosse Developers Meeting calendar to your Google Calendar, <a href="placeholderForLinkToSubscribeToCalendar">click here</a></p>
```

Comment:
Can you remove this placeholder for now? And then I'll approve and merge. 🙂

Reply:
Oh yeah! I'll do that right now.